doc/theoretical_description_regression.rst

```diff
@@ -214,6 +214,32 @@ In practice, this method results in wider prediction intervals, when the
 uncertainty is higher, than :math:`CV+`, because the models' prediction spread 
 is then higher.
 
+
+8. The conformalized quantile method
+====================================
+
+The conformalized quantile method allows for a better interval width in cases
+of heteroscedasticity data. It uses quantile regressors wiht different quantile
+values to estimate the prediction bounds and the residuals of these methods is
+used to create the guaranteed coevrage value.
+
+.. math:: 
+
+    \hat{C}_{n, \alpha}^{\rm conformalized quantile}(X_{n+1}) = 
+    [\hat{q}_{\alpha_{lo}}(X_{n+1}) - Q_{1-\alpha}(E, \mathcal{I}_2),
+    \hat{q}_{\alpha_{hi}}(X_{n+1}) + Q_{1-\alpha}(E, \mathcal{I}_2)]
+
+Where :math:`Q_{1-\alpha}(E, \mathcal{I}_2) := (1-\alpha)(1+1/\abs(\mathcal{I}_2))`-th
```

Comment:
Can you check that the code is correct ? The documentation seems to have failed here : https://mapie--167.org.readthedocs.build/en/167/theoretical_description_regression.html

Reply:
done